content/docs/command-reference/dag.md

```diff
@@ -21,19 +21,19 @@ show the full project DAG.
 ### Directed acyclic graph
 
 A data pipeline, in general, is a series of data processing
-[stages](/doc/command-reference/run) (for example, console commands that take an
-input and produce an outcome). The connections between stages are formed by the
-<abbr>output</abbr> of one turning into the <abbr>dependency</abbr> of another.
-A pipeline may produce intermediate data, and has a final result.
+[stages](/doc/command-reference/stage) (for example, console commands that take
```

Comment:
OK I see we make more run-related changes here than in https://github.com/iterative/dvc.org/pull/3223/files#diff-7b6ee3d8c0ebb9e2a9dcf9077d16fda46133fb069f3879a3c38b0653fe00e850 but updating these links seems like a separate task. Let's create an issue to decide where should the "stage" definition be (probably a guide?) and then update links throughout.

Reply:
Fine. Closed.